docs/boards/Board - CrazyBeeF3FR.md

```diff
@@ -0,0 +1,74 @@
+# CrazyBee F3 FR
+![CrazyBee F3 FR front](https://raw.githubusercontent.com/githubDLG/betaflight/master/docs/boards/CrazyBeeF3FRtop.jpg)
+![CrazyBee F3 FR back](https://raw.githubusercontent.com/githubDLG/betaflight/master/docs/boards/CrazyBeeF3FRbottom.jpg)
```

Comment:
Now that the image is in the same repository than md file, you can use relative url. I've seen that by now, they are in the same folder (I don't know if this is a good idea), but if it is, the image can be referenced by the name only, for example: 
```
![CrazyBee F3 FR front](CrazyBeeF3FRtop.jpg)
```
In your file it references to your repository.

Reply:
seems to be worked:rolling_on_the_floor_laughing: